src/commands/install.yml

```diff
@@ -25,6 +29,11 @@ parameters:
     type: string
     default: "v1"
     description: Change the default node cache version if you need to clear the cache for any reason.
+  include-branch-in-node-cache-key:
+    type: boolean
+    default: false
+    description: >
```

Comment:
Is there a difference here using the ">" or "|" character? We should make them consistent if not.